src/pages/NewAccount/Intro/Intro.scss

```diff
@@ -0,0 +1,22 @@
+@import "~@aeternity/aepp-components/dist/variables.scss";
+@import "../../../text.style.scss";
+
+.intro.mobile-page .panel {
+  .content {
+    h1 {
+      @include titlelarge;
+      width: 90%;
+      font-size: 35px;
+    }
+    p {
+      @include paragraphnormal; 
+      width: 90%;
+    }
```

Comment:
Styles for `h1` and `p` shouldn't be duplicated in every component. I'm proposing to fix it in #158, https://github.com/aeternity/aepp-identity/pull/158/files#diff-682061e114317cf56a4c6edcfd67960bR16.